databricks_sql_connector_core/src/databricks/sql/auth/authenticators.py

```diff
@@ -25,12 +31,10 @@ class CredentialsProvider(abc.ABC):
     for authenticating requests to Databricks REST APIs"""
 
     @abc.abstractmethod
-    def auth_type(self) -> str:
-        ...
```

Comment:
Not related to this PR and shouldn't be fixed here, but I've already seen this so much all over our codebase so far. Even though ellipsis literal _can_ be used as a function body, it's not a replacement for `pass`. `pass` actually creates function with no body, while ellipsis literal creates a function with body which consists of that ellipsis literal. It's not any different from using any other literal for this purpose, e.g. `def f(): "hello, world"`. I think after we're done with this splitting, we should go over the code and replace these with `pass`